app/bundles/DynamicContentBundle/Form/Type/DynamicContentType.php

```diff
@@ -73,6 +168,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('isPublished', 'yesno_button_group');
 
         $builder->add(
+            'isCampaignBased',
```

Comment:
If not campaign based, how is this DWC item requested without a slot name? Say if I have on my wordpress site three areas that I want content served? Currently campaign DWC is requested by a slot name. If no campaign DWC is found, how does Mautic know what to serve back for the requested slot?

Reply:
This is fixed in the latest code. A slot name is used

Reply:
Fixed